src/nvim/eval.c

```diff
@@ -24033,6 +24050,7 @@ static inline TerminalJobData *common_job_init(char **argv,
   }
   Process *proc = (Process *)&data->proc;
   proc->argv = argv;
+  proc->quote_argv = quote_argv;
```

Comment:
I understood the meaning of the flag in reverse. The code I intended was ```proc->quote_argv = *p_sxq == NUL ? true : quote_argv;```. This should not affect ```system([])/jobstart ([])```/etc.

How about making ```if (!proc->quote_argv && *p_sxq != NUL) {``` in libuv_process.c rather than ```!shellcmd || *p_sxq == NULL``` in multiple places?